microsoft-edge/webview2/release-notes.md

```diff
@@ -105,27 +105,132 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
 
 
 <!-- ====================================================================== -->
-## 1.0.2045.28
+## 1.0.2106-prerelease
 
-Release Date: September 18, 2023
+Release Date: September 19, 2023
 
-[NuGet package for WebView2 SDK 1.0.2045.28](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2045.28)
+[NuGet package for WebView2 SDK 1.0.2106-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2106-prerelease)
 
-For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 117.0.2045.28 or higher.
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 119.0.2106.0 or higher.
 
 
+#### Experimental features
+
+
+<!-- ------------------------------ -->
+* For WinRT, options have been added to manage custom scheme registration when creating a `CoreWebView2Environment`:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+n/a; WinRT only
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* `CoreWebView2CustomSchemeRegistration` Class
+    * [CoreWebView2CustomSchemeRegistration.AllowedOrigins Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2customschemeregistration?view=webview2-winrt-1.0.2106-prerelease&preserve-view=true#allowedorigins)
+    * [CoreWebView2CustomSchemeRegistration.SchemeName Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2customschemeregistration?view=webview2-winrt-1.0.2106-prerelease&preserve-view=true#schemename)
+* `CoreWebView2EnvironmentOptions` Class
+    * [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.2106-prerelease&preserve-view=true#customschemeregistrations)
+
+##### [Win32/C++](#tab/win32cpp)
+
+n/a; WinRT only
+
+---
+
+
+<!-- ------------------------------ -->
+
+* The `GetProcessExtendedInfo` API provides a snapshot collection of all actively running frames in the associated renderer process. This provides the granularity needed for the host application to know which part of WebView2 is consuming resources, such as memory or CPU usage.

```

Comment:
```suggestion
* The `Frame Process Info` API provides a snapshot collection of all actively running frames in the associated renderer process. This provides the granularity needed for the host application to know which part of WebView2 is consuming resources, such as memory or CPU usage.
```
@victorhuangwq @peiche-jessica - change name of broad API?
Not sure if we should call it "the GetProcessExtendedInfo API", given that the .NET tab & WinRT tab currently don't list any API type or member called GetProcessExtendedInfo(s).  Seems like either those tabs should add those API Ref links that only Win32 lists (including CoreWebView2Environment.GetProcessExtendedInfos), or, don't call it "the GetProcessExtendedInfo API".

.NET tab just lists the following; no word "get":
* CoreWebView2ProcessExtendedInfo
   * CoreWebView2ProcessExtendedInfo.AssociatedFrameInfos
   * CoreWebView2ProcessExtendedInfo.ProcessInfo

 Rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes?branch=pr-en-us-2838&tabs=win32cpp#experimental-features

Reply:
changed to F P I